src/components/bulkConcepts/component/SearchBar.jsx

```diff
@@ -1,9 +1,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import { Button } from 'reactstrap';
 
-const SearchBar = ({ handleChange, searchInput, handleSearch }) => (
+const SearchBar = ({
+  fetchAll, handleChange, searchInput, handleSearch,
+}) => (
```

Comment:
I think such formatting should be handled on another ticket to remove ESlint warnings.

Reply:
@Tittoh I would think for a change like this where I haven't necessarily written new code is fine?

Reply:
@AGMETEOR But it is a change that appears on the PR and it doesn't affect the task at hand.

Reply:
@Tittoh I just think that's an Linter issue... otherwise we are going to be pushing code with warnings from the Linter... and overtime they will accumulate...

Reply:
@Tittoh by the way some linting rules won't even let the application run if the warnings are not resolved. So, in such as case, should we still push such code and let it get merged?

Reply:
This will also add more files changed on the PR. Since some already exist, a ticket can be created to fix all that exist and add ESlint rules on travis to prevent new ones from arising.
If an error exists, the build should stop there.

Reply:
@Tittoh let me create that ticket.

Reply:
@dkayiwa it's [here](https://issues.openmrs.org/browse/OCLOMRS-368)

Reply:
Can you fill its priority and fix version?

Reply:
@dkayiwa I have filled in the information.